hugo.yaml

```diff
@@ -191,8 +191,17 @@ module:
       target: content/community/roadmap.md
     - source: static
       target: static
+    - source: content-modules/semantic-conventions/schemas
+      target: static/schemas
+      # TODO: drop the lines marked below once
+      # https://github.com/open-telemetry/opentelemetry.io/issues/2721
+      # is fully resolved.
+      # [DROP-FROM-HERE
+      excludeFiles: [1.21.0]
     - source: content-modules/opentelemetry-specification/schemas
       target: static/schemas
+      includeFiles: [1.21.0]
```

Comment:
We should NOT be serving a `1.21.0` right now, from either location.

Reply:
Ok, that's news to me. It has been served since the OTel spec 1.21.0 was released in May. Back then the website served whatever schemas are found under the OTel spec's `schemas` folder and no one from the specs team has informed us otherwise.

If you think that this is a problem, please open an issue over this repo stating that the schema should be removed and cc @tigrannajaryan and the rest of the OTel spec maintainers. Thanks!

Reply:
Ah, ok, I just saw: https://github.com/open-telemetry/opentelemetry-specification/issues/3572.

I'll create a corresponding issue here, and a PR to remove the 1.21.0 schema.

Reply:
Here's the PR:

- #2924